### DISCUSSION TITLE
蓝灯开始耍流氓了吗？

### DISCUSSION THREAD

**Discussion by David68326** [getlantern/lantern-forum-cn#741] [General]
用不了不退款还删帖子，流氓本性，我是两年的会员，我会经常光顾的

**Comment by MaxMKMax** [comment]
@David68326 您好. 请问您那里是遇到什么问题? 您告诉我您的问题我帮您解决. 我是不记得您是哪个用户的问题了. 不好意思. 很抱歉.

**Comment by MaxMKMax** [comment]
@David68326 请您息怒. 有什么是我们和您说声抱歉. 是您的情况无法退款吗? 之前贴出来的是我们工作人员的聊天记录和系统本身显示无法退款. 

因为我们用的是第三方系统, 过了一定时间就不能操作了. 然而我们的购买协议有写购买后是不退款. 
只是我们内部有放松条款还是 2 个月内是给退款. 所以才有退款这件事情.

**Reply by David68326** [reply]
能正常使用当然无需退款，问题是你们的软件无法正常使用造成的，付款的时候是你给我的支付宝链接，什么时候给过我协议？再说了，哪个正经的公司会在协议里写卖给你的是垃圾也不能退款？而且我已经告诉过你了，第三方平台已经明确答复了，一年之内的订单都可以退款，只需要你们提出申请即可，这个还要我再说多少次？
你发给我的聊天记录和截图用的是别人的订单，故意把订单编号遮掩起来，但仔细看还是能看出并不是我的订单号。发别人的东西给我想证明什么？和我的订单有什么关系，为什么不用我的订单号去操作？
你们一直说要改进让我们等新版本，但结果呢？从2月份到现在半年了，还不是一样不能正常使用？你自己是不是可以看看论坛里有多少不能正常使用的，有多少要求退款的？退款的都是被你们一拖再拖，然后说太久了不能办了，真是可笑！
已经有人开始组团在两个平台上投诉你们了，我买了两年的会员，如果你们不能退款，我会一直坚持的，反正有的是时间。
会员邮箱：815357956@qq.com，你们可以查查我有多少个月没有用蓝灯了。

**Comment by MaxMKMax** [comment]
![image](https://github.com/user-attachments/assets/d8f1523b-236e-43f0-ad4a-7b589ec8a941)
@David68326 没有人要故意藏您的订单. 我收到的截图的确如此. 而且也和你说过了, 你可以和银行那里申请退款.

支付部门的回答: 不确定我们是否可以做些什么，但如果他们对自己的经历真的感到失望，他们也许可以通过自己的银行对收费提出异议？

我这里能帮您做的只是在幫你問看还有没有别的方式可以退款. 我再帮你问看. 你给我几天的时间.

**Comment by MaxMKMax** [comment]
@David68326 你在提供我你的支付单据吧. 我帮你再申请.

**Comment by David68326** [comment]
好的，我再发一次支付单据，如果又说看不到，可以查看原来附有支付单据的帖子#465。
我的商家订单号是src_1OgUNFDTyZkPGbMkRSI1NJ3B
你的聊天记录截图中的订单号并不是我的。你给我的第三方平台是支付宝，我已经说过很多次了，支付宝官方回复是一年内的订单都可以办理退款，但是要收款方也就是你们向平台提出退款申请。我是真不理解，你们知道怎么收款但不知道怎么退款吗？如果真的不懂，请直接咨询支付宝官方客服，他们会教你们怎么做的。

---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年8月1日(周四) 凌晨1:09
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 蓝灯开始耍流氓了吗？ (Discussion #741)




 
@David68326 你在提供我你的支付单据吧. 我帮你再申请.

**Reply by MaxMKMax** [reply]
@David68326 我们这里帮您问着了. 下周给您答复. 谢谢您的耐心等待.

**Reply by David68326** [reply]
@Mike777Khoo 又一周过去了，还在等你的回复

**Reply by David68326** [reply]
@Mike777Khoo 请问怎么回事？为什么两周了还不回复？

**Comment by David68326** [comment]
---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年8月1日(周四) 凌晨1:09
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 蓝灯开始耍流氓了吗？ (Discussion #741)




 
@David68326 你在提供我你的支付单据吧. 我帮你再申请.
